static/js/hooks/useAllWebsites.ts

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react"
+import { useSelector } from "react-redux"
+import { useRequest } from "redux-query-react"
+import { websiteListingRequest } from "../query-configs/websites"
+import {
+  allWebsitesSelector,
+  getWebsiteListingCursor
+} from "../selectors/websites"
+import { Website } from "../types/websites"
+import { WEBSITE_CONTENT_PAGE_SIZE } from "../constants"
+
+/**
+ * A rather brute-force way to fetch all the websites, to be able to
```

Comment:
This is fetching more than 150 requests on my local machine. Is there a way to pass in (or just hardcode) a larger value for `limit`?

Reply:
ahh yeah I don't have a realistic set of courses locally I think, haha

I'll look into this stuff tomorrow